setup.py

```diff
@@ -49,7 +49,7 @@ def get_version():
             'autopep8',
             'mock; python_version < "3.3"',
             'mypy; python_version >= "3.3"',
-            'pep8',
+            'pycodestyle == 2.2.0',
```

Comment:
Why the specific version? Might lead into troubles if a consumer application requires a newer version.